core/domain/stats_domain.py

```diff
@@ -584,13 +644,36 @@ def __init__(self, answer, frequency):
         self.frequency = frequency
 
     def to_raw_type(self):
+        """Returns dict for specific answer in its raw type.
+
+        Returns:
+            dict(str, *). The specific answer dict in the following format:
+            {
+                answer: The answer submitted by the learner.
```

Comment:
'answer': *. The answer ...
'frequency': int. The number of ...

Use '...' to distinguish strings from variable names.

Reply:
Done!